backend/src/Edna/Web/Swagger.hs

```diff
@@ -52,13 +52,9 @@ type SwaggerUI =
 type WithSwaggerUI api = api :<|> SwaggerUI
 
 -- | Attach an UI serving given documentation to the given server.
-withSwaggerUI
-  :: Proxy api
-  -> S.Swagger
-  -> Server api
-  -> Server (WithSwaggerUI api)
-withSwaggerUI _ swagger server =
-  server :<|> swaggerSchemaUIServer swagger
+withSwaggerUI :: Proxy api -> O.OpenApi -> Server api
+              -> Server (WithSwaggerUI api)
```

Comment:
I don't know if we have any formal strict rules in style guide about indentation in this case, but you can see that in all examples in style guide we use fixed indentation. Example:
![image](https://user-images.githubusercontent.com/2556409/117432729-5a69ad00-af33-11eb-8846-561359c9196c.png)

As you can see, there are always 2 spaces no matter what the length of `putValueInState` is. The advantage of this approach is that if we rename `putValueInState` we won't need to modify lines of code below `putValueInState`.

In case of our code (`withSwaggerUI`) this indentation forces us to increase/decrease indentation on this line if we rename `withSwaggerUI`.

Reply:
I've got. The reason I've used such indention, that if I put `:: ...` on newline, vscode stops to highlight function name ( but this is issue to plugin
I'll update it